notebooks/Media-Pretrained/02_Training_Layer_Recap.ipynb

```diff
@@ -111,9 +87,1051 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Retrieve your automated deployment variables\n",
+    "# How to train your Use Case Optimized Recommenders and Solution Versions\n",
```

Comment:
Use Case Optimized Recommenders is not a proper name.  Suggest:

"use case-optimized recommenders" OR "use-case-optimized recommenders"